griptape/rules/base_rule.py

```diff
@@ -5,10 +5,12 @@
 
 from attrs import define, field
 
+from griptape.mixins.serializable_mixin import SerializableMixin
 
-@define(frozen=True)
```

Comment:
Unfortunate but unavoidable without massive refactors.

Reply:
Would love a little more info on the thought process. If the goal is to make `value` and `meta` immutable is there any other ways? Like override the setter?

Reply:
The change comes from pyright's note that:
```A frozen class cannot inherit from a class that is not frozen [reportGeneralTypeIssues]```

Making `SerializableMixin` frozen would be a massive (maybe unfeasable) change.